src/PrestaShopBundle/Controller/Admin/Improve/Design/MailThemeController.php

```diff
@@ -356,6 +360,42 @@ public function sendTestMailAction($theme, $layout, $locale, $module = '')
         return $this->redirectToRoute('admin_mail_theme_preview', ['theme' => $theme]);
     }
 
+    /**
+     * @param Request $request
+     *
+     * @return \Symfony\Component\HttpFoundation\RedirectResponse
+     */
+    public function translateBodyAction(Request $request)
+    {
+        $translateMailsBodyForm = $this->createForm(TranslateMailsBodyType::class);
+        $translateMailsBodyForm->handleRequest($request);
+
+        if (!$translateMailsBodyForm->isSubmitted() || !$translateMailsBodyForm->isValid()) {
+            $this->addFlash(
+                'error',
+                $this->trans(
+                    'Cannot send translate body contents for e-mail',
```

Comment:
What is it supposed to mean?

Reply:
Oops, my bad it was a typo. I replaced it with `Cannot translate body contents for e-mail` is t clearer?

Reply:
Please write _Cannot translate emails body contents_ instead.

Reply:
Alright, done